File: webserv_correc/html/cgi-bin/displayenv.php
<?php
	$env = getenv();
?>
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>WEBSERV CGI</title>
</head>
<body>
<h1>Webserv CGI environnement variables</h1>
<?php
	foreach ($env as $key => $value) {
		echo "<p><strong>$key =></strong> \"$value\"</p>\n";
	}
?>
<p><strong>© BB & JN :-)</strong></p>
</body>
</html>

File: html/cgi-bin/displayenv.pl
#!/usr/bin/env perl

print "<!DOCTYPE html>\n";
print "<html>\n";
print "<head>\n";
print "<meta charset='UTF-8'>\n";
print "<title>Variables d'environnement</title>\n";
print "</head>\n";
print "<body>\n";
print "<h1>Voici les variables d'environnement envoyées au CGI : </h1>\n";

foreach $key (sort(keys %ENV)) {
  print "<strong>$key</strong> = $ENV{$key}<br>\n";
}
print "</body>\n";
print "</html>\n";
